pages/common/git-reflog.md

```diff
@@ -0,0 +1,11 @@
+# git reflog
+
+> Show when the refs were updated in local repository.
+
+- View reflog:
+
+`git reflog`
+
+- View N latest entries in reflog:
+
+`git reflog -n {{number_of_entries}}`
```

Comment:
If you mention N in the description, the example should also have N. Otherwise, you can mention an actual no. too.

Reply:
Done!